src/main/java/org/example/attorney/AttorneyParser.java

```diff
@@ -0,0 +1,33 @@
+package org.example.attorney;
+
+import org.example.attorney.*;
+import org.jsoup.*;
+import org.jsoup.nodes.*;
+
+import java.io.*;
+import java.util.*;
+import java.util.ArrayList;
+import java.util.Collections;
+
+public class AttorneyParser {
+    public static List<Attorney> parseAttorneys(List<String> attorneyLinks) throws IOException {
```

Comment:
using static on methods is procedural style not OOP
```suggestion
    public List<Attorney> parseAttorneys(List<String> attorneyLinks) throws IOException {
```